### DISCUSSION TITLE
几个后续版本的优化建议

### DISCUSSION THREAD

**Discussion by lopins** [1Panel-dev/1Panel#5780] [General]
1. 既然是以docker为核心，非宝塔面板那种全局环境，网站下方的运行环境创建觉得很鸡肋，不如将这个取消，改到新加网站时即时部署，方便各个项目配置对应依赖扩展; php扩展配置简化，目前不适于小白用户使用.
2. docker网站应用增加绑定域名、证书等（和普通网站项目一样），如: adminer、lobehub、pma等等.
3. docker增加可选自动同步拉取.

**Comment by wanghe-fit2cloud** [comment]
感谢建议。
